### ISSUE TITLE
Render issue when formule is inside { and }

### ISSUE THREAD

**Issue by scaljeri** [Khan/KaTeX#220]
I have the following formula:

    {25*0.008^x, 25*(1/125)^x}

With KaTeX (v0.3.0) this formula render with the enclosing `{` and `}` But when I view this formula with MathJax or some other online latex render tool it shows this formula without the brackets. Is this a (known) bug or is there something I did wrong

[MathJax online renderer](http://bit.ly/1wTAG0F)
[My very own KaTex renderer](http://scaljeri.github.io/math-editor/) (goto the second tab and you can past the formula in the textarea)

**Comment by PeterBocan** [comment]
I think you have to escape these special characters as follows:

      \{ 25*0.008^x, 25*(1/125)^x \}

**Comment by xymostech** [comment]
It looks like you're passing your input through AsciiMath before turning it into KaTeX, which is why it isn't working for you. AsciiMath is outputting the formula:

    {\left\lbrace{25}\cdot{0.008}^{{x}},{25}\cdot{\left(\frac{{1}}{{125}}\right)}^{{x}}\right\rbrace}

which is being correctly rendered (you can try it at http://khan.github.io/KaTeX/). I'm not sure how AsciiMath works, but that is why you are seeing the extra `{` characters.

Also, it looks like you're including KaTeX v0.1.1 on that page, not v0.3.0.

**Comment by scaljeri** [comment]
What do you mean with `through AsciiMath` ? Like: I convert `AsciiMath` to `Latex` ? or that I try to convert `Latex` to `Latex` using an `AsciiMath` library as if the input is actually AsciiMath (but isn't) ? Because something like this is possible, because I get the formulas in any format. 

So how can you tell I do something with AsciiMath ? Are the curly brackets invalid LaTeX ?

**Comment by xymostech** [comment]
Correct, you are using AsciiMath on your website. If you type in `{25*0.008^x, 25*(1/125)^x}` into that website, it says that the input format is AsciiMath, which means that it runs your math through the AsciiMath converter before it gets processed by KaTeX. (I checked this by setting a breakpoint in the `katex.render` function, and ensuring that that the LaTeX that is sent to KaTeX isn't `{25*0.008^x, 25*(1/125)^x}`, but the expression I pasted above).

If you try typing in that expression somewhere that only renders KaTeX and doesn't do the AsciiMath transform, like at http://khan.github.io/KaTeX/, you won't see the `{` and `}` (but you'll also see `*` instead of `\cdot`, and a literal `1 / 125` instead of a fraction, because those are both transformations that AsciiMath does).
